src/content/reference/helm-chart-values.mdx

```diff
@@ -151,6 +151,18 @@ auth:
 
 The `organization` field is optional. Only members of the organization will be allowed to log in into your Okteto instance. An empty `organization` field permits any user to log in.
 
+If you want to provide access to only select users of your organization, you can additionally use the `allowList` key to list the GitHub accounts you want to allow.
```

Comment:
`allowList` is for any user not belonging to the organization, we cannot filter out members of the organization. To do that, you wouldn't define any organization and list all the allowed users in `allowList`

Reply:
```suggestion
You can provide access to select users with the `allowList` key to list the GitHub accounts you want to allow. This feature only works for listing individual GitHub accounts and does not work in conjunction with GitHub organizations.
```

Suggested change based on what I'm understanding here 😄